### ISSUE TITLE
Remove 'All rights reserved' from the copyright headers

### ISSUE THREAD

**Issue by xtqqczze** [microsoft/vscode#96747]
Since July 2019 Microsoft no longer recommend the term All Rights Reserved to be in MIT license copyright headers.

The headers should be replaced with:

> // Copyright (c) Microsoft Corporation.
> // Licensed under the MIT License.

<del>`LICENSE.txt` could also be updated with the standard wording from https://github.com/microsoft/referencesource/blob/master/LICENSE.txt</del>

Previous vscode issues:
* <del>https://github.com/microsoft/vscode/pull/62074</del>
* https://github.com/microsoft/vscode/issues/75060

References:
* https://github.com/microsoft/react-native-windows/issues/3933
* https://github.com/PowerShell/PowerShell/pull/12190

**Comment by xtqqczze** [comment]
@vscodebot not a duplicate

**Comment by chrisdias** [comment]
agreed.  i will update the `LICENSE.txt` file through https://github.com/microsoft/vscode/pull/62074 and then we need to update the sources to say:

``` 
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation.
 *  Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
```

assigning to @kieferrm for the source changes.

**Comment by xtqqczze** [comment]
@kieferrm I could make the source changes and you could review?

**Comment by xtqqczze** [comment]
@chrisdias @kieferrm I have made part 1 of changes in #104498

**Comment by xtqqczze** [comment]
@kieferrm What is the best way to move forward on this?
